pkg/trace/obfuscate/sql.go

```diff
@@ -67,6 +71,12 @@ func (f *discardFilter) Filter(token, lastToken TokenKind, buffer []byte) (Token
 			// and will continue to be discarded until we find the corresponding
 			// closing bracket counter-part. See GitHub issue DataDog/datadog-trace-agent#475.
 			return FilteredBracketedIdentifier, nil, nil
+		} else if token == '(' {
```

Comment:
We can reduce to one single `if`?
```suggestion
		} else if token == '(' && config.HasFeature("sql_reduce") {
```